packages/expo-updates/android/src/main/java/expo/modules/updates/DisabledUpdatesController.kt

```diff
@@ -26,7 +34,15 @@ class DisabledUpdatesController(
   private val context: Context,
   private val fatalException: Exception?,
   private val isMissingRuntimeVersion: Boolean
-) : IUpdatesController {
+) : IUpdatesController, UpdatesStateChangeEventSender {
+  private var reactNativeHost: WeakReference<ReactNativeHost>? = if (context is ReactApplication) {
+    WeakReference(context.reactNativeHost)
+  } else {
+    null
+  }
```

Comment:
The weak reference has been there since this was originally written (copied from Enabled controller which is the modern incarnation of the original). I'll look into this in a follow-up!